vumi_http_retry/tests/test_redis.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+from twisted.trial.unittest import TestCase
+from twisted.internet.defer import inlineCallbacks
+
+from vumi_http_retry.tests.redis import create_client, zitems
+
+
+class TestRedis(TestCase):
+    @inlineCallbacks
+    def setUp(self):
+        self.redis = yield create_client()
+
+    @inlineCallbacks
+    def tearDown(self):
+        yield self.redis.delete('foo')
```

Comment:
Txredis isn't adding a prefix, but a common pattern we've used in Vumi is to store everything under a set prefix (e.g. `go.` or `vumi.test.`) and then clear everything under that when the test ends (`keys = redis.keys('vumi.test.*'); for k in keys: redis.delete(k)`).

Happy for that to be left out of this PR though and sorted out later if needed.